webui/react/src/pages/TaskList.tsx

```diff
@@ -249,6 +104,30 @@ const TaskList: React.FC = () => {
     return false;
   }, [ selectedTasks ]);
 
+  const filterConfigMap = useMemo(() => {
+    return filterKeys.reduce((acc, key) => {
+      const config = settingsConfig.settings.find(config => config.key === key);
+      return { ...acc, [key]: config };
+    }, {} as Record<keyof Settings, SettingsConfigProp>);
+  }, []);
+
+  const filterCount = useMemo(() => {
```

Comment:
suggestion: similar to the comment below. If the useSettings hook could give you a list of "active" settings IMO that could make this portion reusable without specializing it to filters.

Reply:
totally makes sense!